app/views/eventDetails/regTypes.html

```diff
@@ -147,9 +147,9 @@ <h4 translate>Payment</h4>
                 <p class="text-muted"><i class="fa fa-info-circle"></i> <translate>If blank, full cost will be required at time of registration.</translate></p>
               </div>
 
-              <div class="pt12" ng-show="!conference.requireLogin || !anyPaymentMethodAccepted(type)">
+              <div class="pt12" ng-show="!(conference.relayLogin || conference.facebookLogin) || !anyPaymentMethodAccepted(type)">
```

Comment:
It may be a good idea to move this check `conference.relayLogin || conference.facebookLogin` into an attribute on conference.